src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -472,4 +475,20 @@ public void nextPreviewStyle() {
     public void previousPreviewStyle() {
         this.previewTabs.forEach(OffersPreview::previousPreviewStyle);
     }
+
+    public static boolean checkLastTextField(TabPane tabs, TextField textField) {
```

Comment:
Please make this non-static.

Reply:
Hey,
Sorry, I'm not sure how. I believe I can make checkLastTextField non-static and use the tabbed TabPane in the EntryEditor instead. But in doing this, I would need to create a new instance of EntryEditor in EditorTextField which will require the following arguments: EntryEditor(LibraryTab libraryTab, UndoAction undoAction, RedoAction redoAction). -> which I'm not sure how to get from EditorTextField.
Please guide me how I could do this.
 Thankyou!

Reply:
If im calling the non-static method `checkLastTextField` that's in `EntryEditor`, that uses `private TabPane tabbed` from `EditorTextField`. It requires me to create a new instance of `EntryEditor` in `EditorTextField`. I believe

Reply:
If you create a new instance of an EntryEditor your changes would not affect the one we are displaying. There is only one EntryEditor. You probably have to inject a reference to the object you need.
But I believe there is a more serious architectural flaw, if you are trying to get a reference to to the big container node.
Why would you need the entry editor? Why would you need the TabbedPane? And what exactly do you need from the TabbedPane?

Reply:
You can `inject` a method `selectNextTab` or sthg as a functional interface if really necessary.

Reply:
I need the TabPane to get the last Field of a tab and to select the next Tab